dmp/user-story-single-dmp-connections.ipynb

```diff
@@ -166,11 +166,6 @@
     "          }\n",
```

Comment:
<p>I think you would still need to remove filter_people from inside the create_node execution. Here it will filter by name type for three arrays of results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/datacite/pidgraph-notebooks-python/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes, forgot to apply that to the changes here. I have removed that function